manifests/sudoers.pp

```diff
@@ -0,0 +1,142 @@
+# This class provides some useful aliases that many people have wanted
+# to use over time.
+#
+# None of this is mandatory and all can be changed via the different
+# parameters.
+#
+# Each section simply adds the entry to the sudoers file by joining
+# the array together appropriately.
+#
+# @author Trevor Vaughan <tvaughan@onyxpoint.com>
+#
+class simp::sudoers (
```

Comment:
Make the aliases a subclass and include if a selector is true. Some people don't want them and we don't actually use them by default.